apps/readest-app/src/app/reader/components/settings/MiscPanel.tsx

```diff
@@ -71,7 +74,7 @@ const MiscPanel: React.FC<{ bookKey: string }> = ({ bookKey }) => {
       getView(bookKey)?.renderer.removeAttribute('animated');
     }
     // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [animated]);
+  }, [animated,isDisableClick]);
```

Comment:
Better to have two useEffect blocks, one for `animated` and one for `isDisableClick`.